src/js/flights/flightView.js

```diff
@@ -19,8 +17,6 @@ const flightView = flightData => {
 };
 
 const getFlight = async city => {
-  try {
-    await flightView(await getFlightData(city));
-  } catch (err) {}
+  await flightView(await getFlightData(city));
```

Comment:
czekaj, co? :eyes:

Reply:
Zostawiłam jak było, jedynie usunęłam try/catch bo miałam problem wyłapać błąd z Promise.all(). Jeszcze raz sprawdzę to.
Jeśli zapisze "podwójny await" w taki sposób, to będzie ok? Bo nie wiem o co chodzi dokładnie - domyślam się jedynie 😄 
```
const flightsData = await getFlightData(city);
flightView(flightsData)
```

Reply:
zmieniłam na tak jak wyżej napisałam, ale try/catch z Promise.all się nie lubiły.